source/views/BeginnerView.mc

```diff
@@ -36,7 +28,8 @@ class BeginnerViewDelegate extends Ui.BehaviorDelegate {
 
 	function manageChoice(player) {
 		match.begin(player);
-		Ui.switchToView(new MatchView(), new MatchViewDelegate(), Ui.SLIDE_IMMEDIATE);
+		var view = new MatchView();
+		Ui.switchToView(view, new MatchViewDelegate(view), Ui.SLIDE_IMMEDIATE);
```

Comment:
Is this required? Should all "Delegates" be constructed with their view as first parameter? I don't think so (see all Garmin samples).

If yes, we should initialize all "Delegates" the same way.